__tests__/integrations.test.js

```diff
@@ -85,25 +88,78 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
-describe('GET /api/articles', () => {
-  test('GET 200: responds with an array of articles', () => {
-    return request(app).get("/api/articles").expect(200).then(({body}) => {
-      expect(body.articles.length).toBe(13)
-      expect(body.articles).toBeSortedBy("created_at", {
-        descending: true
-      })
-      body.articles.forEach((article) => {
-        expect(article).toMatchObject({
-          author: expect.any(String),
-          title: expect.any(String),
-          article_id: expect.any(Number),
-          topic: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-          article_img_url: expect.any(String),
-          comment_count: expect.any(Number),
-        })
-      })
-    })
+describe("GET /api/articles", () => {
+  test("GET 200: responds with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET 200: responds with an array of comments for the given article ID sorted by descending date", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(11);
+        expect(comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 1,
```

Comment:
Well done including this little bit of extra security - a lot of people miss this, but you're absolutely correct in checking whether all the comments have the expected article_id 👍